Adds how to run a collator using docker & system (#106)
moondance-labs/tanssi-docs: .snippets/text/node-operators/optimized-binaries-note.md

!!! note

>     It is recommended that you run the optimized binary versions for either [Skylake](https://www.intel.com/content/www/us/en/products/platforms/details/skylake-u-y.html){target=\_blank} or [Zen3](https://www.amd.com/en/technologies/zen-core){target=\_blank} architectures. They are available in the [releases](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} page.

Reviewer: ```suggestion
    For better performance, it is recommended that you run the optimized binary versions for either Intel's [Skylake](https://www.intel.com/content/www/us/en/products/platforms/details/skylake-u-y.html){target=\_blank} or AMD's [Zen3](https://www.amd.com/en/technologies/zen-core){target=\_blank} architectures. 
```
